integration/operator_test.go

```diff
@@ -17,74 +17,14 @@ import (
 	"github.com/giantswarm/e2e-harness/pkg/framework"
 	"github.com/giantswarm/microerror"
 
+	"github.com/giantswarm/aws-operator/integration/template"
 	"github.com/giantswarm/aws-operator/service"
 	"github.com/giantswarm/aws-operator/service/awsconfig/v2/key"
 )
 
 const (
-	awsOperatorValuesFile  = "/tmp/aws-operator-values.yaml"
-	awsOperatorChartValues = `Installation:
-  V1:
-    Guest:
-      Kubernetes:
-        API:
-          Auth:
-            Provider:
-              OIDC:
-                ClientID: ""
-                IssueURL: ""
-                UsernameClaim: ""
-                GroupsClaim: ""
-      Update:
-        Enabled: ${GUEST_UPDATE_ENABLED}
-    Name: ci-aws-operator
-    Provider:
-      AWS:
-        Region: ${AWS_REGION}
-    Secret:
-      AWSOperator:
-        IDRSAPub: ${IDRSA_PUB}
-        SecretYaml: |
-          service:
-            aws:
-              accesskey:
-                id: ${GUEST_AWS_ACCESS_KEY_ID}
-                secret: ${GUEST_AWS_SECRET_ACCESS_KEY}
-                token: ${GUEST_AWS_SESSION_TOKEN}
-              hostaccesskey:
-                id: ${HOST_AWS_ACCESS_KEY_ID}
-                secret: ${HOST_AWS_SECRET_ACCESS_KEY}
-                token: ${HOST_AWS_SESSION_TOKEN}
-      Registry:
-        PullSecret:
-          DockerConfigJSON: "{\"auths\":{\"quay.io\":{\"auth\":\"${REGISTRY_PULL_SECRET}\"}}}"
-`
-	awsResourceValuesFile  = "/tmp/aws-operator-values.yaml"
-	awsResourceChartValues = `commonDomain: ${COMMON_DOMAIN_GUEST}
-clusterName: ${CLUSTER_NAME}
-clusterVersion: v_0_1_0
-sshPublicKey: ${IDRSA_PUB}
-versionBundleVersion: ${VERSION_BUNDLE_VERSION}
-aws:
-  networkCIDR: "10.12.0.0/24"
-  privateSubnetCIDR: "10.12.0.0/25"
-  publicSubnetCIDR: "10.12.0.128/25"
-  region: ${AWS_REGION}
-  apiHostedZone: ${AWS_API_HOSTED_ZONE_GUEST}
-  ingressHostedZone: ${AWS_INGRESS_HOSTED_ZONE_GUEST}
-  routeTable0: ${AWS_ROUTE_TABLE_0}
-  routeTable1: ${AWS_ROUTE_TABLE_1}
-  vpcPeerId: ${AWS_VPC_PEER_ID}
-`
-	// nodeOperatorChartValues values required by node-operator-chart, the environment
-	// variables will be expanded before writing the contents to a file.
-	nodeOperatorChartValues = `Installation:
-  V1:
-    Secret:
-      Registry:
-        PullSecret:
-          DockerConfigJSON: "{\"auths\":{\"quay.io\":{\"auth\":\"$REGISTRY_PULL_SECRET\"}}}"
-`
+	awsOperatorValuesFile = "/tmp/aws-operator-values.yaml"
+	awsResourceValuesFile = "/tmp/aws-operator-values.yaml"
```

Comment:
JFYI In cloud configs templates we store the path together with content. Uncertain if makes much sense in here.

Reply:
Uncertain. We will learn this along the road.